source/jormungandr/jormungandr/scenarios/distributed.py

```diff
@@ -229,8 +279,16 @@ def call_kraken(self, request_type, request, instance, krakens_call, context):
         """
         try:
             with FutureManager() as future_manager:
-                res = self._compute_all(future_manager, request, instance, krakens_call, context)
-                return res
+                return self._scenario._compute_all(future_manager, request, instance, krakens_call, context)
+        except PtException as e:
+            return [e.get()]
+        except EntryPointException as e:
+            return [e.get()]
+
+    def finalise_journeys(self, request, responses, context, instance, is_debug):
+        try:
```

Comment:
hmmm, i think i wasn't very clear in my previous comment :p, sorry for that,  the exception that may arrive here are no longer PtException (there're no pt computation) or EntryPointException(no entry research either).

I think a standard Exception should be ok...